src/main/pages/workspace-admin/recipes.adoc

```diff
@@ -11,42 +11,56 @@ folder: workspace-admin
 [id="supported-recipe-formats"]
 == Supported Recipe Formats
 
-Depending on the infrastructure, Che supports the following recipes out of the box (default Che build):
+Depending on the infrastructure, Che supports the following default recipe formats:
 
 [cols=",,,,",options="header",]
 |===
-|*Infrastructure* |*Docker Image* |*Dockerfile* |*Composefile* |*Kubernetes yaml*
+|*Infrastructure* |*Docker-formatted container image* |*Dockerfile* |*Composefile* |*Kubernetes YAML*
 |*Docker* |Supported |Supported |Supported |Not supported
 |*OpenShift* |Supported |Not supported |Not supported |Supported
 |===
 
-[id="docker-image"]
-== Docker Image
+[id="docker-formatted-container-image-requirements-and-limitations"]
```

Comment:
Did you search the docs to find out if the previous anchor was used anywhere? I just want to make sure that nothing breaks by accident. :)

Reply:
Great point!  I will do.